mmv1/third_party/terraform/services/storagetransfer/resource_storage_transfer_job.go.tmpl

```diff
@@ -98,6 +120,7 @@ func ResourceStorageTransferJob() *schema.Resource {
 				Optional:      true,
 				MaxItems:      1,
 				ConflictsWith: []string{"schedule"},
+				DiffSuppressFunc: diffSuppressEventStream,
```

Comment:
What is the need of DiffSuppressFunc?

Reply:
Good question! I'll give some context:

When using cross bucket replication feature, i.e specifying `replication_spec`, the backend would setup `event_stream` on it's own, and it can't be configurable via user, Hence showing `computed` behvior, However we can't mark it `computed` as the field is configurable while using `transfer_spec`.

So first time when you do `terraform apply`,  the `event_stream` would be stored in file, and in next `terraform plan` it would show a diff `SOMETHING -> ""`

One alternative to this is to not store `eventStream` at all in `Read` while using `replication`.

Reply:
From the context, Can we use Optional+Computed and ConflictsWith(event_stream and replication_spec)?

Reply:
Keeping it `Optional+Computed` would show `event_stream (known after apply)` despite not specifying `event_stream` while working with `transfer_spec`.

But i think, it should be fine - I've updated code to reflect the same!

Reply:
Also please consider a case when user removes `event_stream` from the config expect it to be removed from the resource. If the attribute is computed then terraform won't consider this change as necessary and won't show any diff. If that is expected then this solution should work otherwise we need to verify if empty list can be used as not-present indicator or not.

Reply:
Based on the offline conversation, we're moving forward with `diffSupressFunc` as written earlier.